src/NCronJob/Registry/IInstantJobRegistry.cs

```diff
@@ -55,101 +65,168 @@ void RunScheduledJob<TJob>(TimeSpan delay, object? parameter = null, Cancellatio
 
-    /// <summary>
-    /// Runs a job that will be executed at <paramref name="startDate"/>.
-    /// </summary>
-    /// <param name="startDate">The starting point when the job will be executed.</param>
-    /// <param name="parameter">An optional parameter that is passed down as the <see cref="JobExecutionContext"/> to the job.</param>
-    /// <param name="token">An optional token to cancel the job.</param>
-    void RunScheduledJob<TJob>(DateTimeOffset startDate, object? parameter = null, CancellationToken token = default)
-        where TJob : IJob;
-
     /// <summary>
     /// Runs a job that will be executed at the given <paramref name="startDate"/>.
     /// </summary>
     /// <param name="jobDelegate">The delegate to execute.</param>
     /// <param name="startDate">The starting point when the job will be executed.</param>
     /// <remarks>
     /// The <paramref name="jobDelegate"/> delegate supports, like <see cref="ServiceCollectionExtensions.AddNCronJob"/>, that services can be retrieved dynamically.
-    /// Also the <see cref="CancellationToken"/> can be retrieved in this way.
+    /// Also, the <see cref="CancellationToken"/> can be retrieved in this way.
     /// </remarks>
     void RunScheduledJob(Delegate jobDelegate, DateTimeOffset startDate);
+
+    /// <summary>
+    /// Runs an instant job to the registry, which will be executed even if the job is not registered and the concurrency is exceeded.
+    /// <param name="parameter">An optional parameter that is passed down as the <see cref="JobExecutionContext"/> to the job.</param>
+    /// <param name="token">An optional token to cancel the job.</param>
+    /// </summary>
+    /// <remarks>
+    /// This is a fire-and-forget process, the Job will be run immediately in the background. The contents of <paramref name="parameter" />
+    /// are not serialized and deserialized. It is the reference to the <paramref name="parameter"/>-object that gets passed in.
+    /// </remarks>
+    /// <example>
+    /// Running a job with a parameter:
+    /// <code>
+    /// instantJobRegistry.RunInstantJob&lt;MyJob&gt;(new MyParameterObject { Foo = "Bar" });
+    /// </code>
+    /// </example>
+    void ForceRunInstantJob<TJob>(object? parameter = null, CancellationToken token = default)
+        where TJob : IJob;
+
+    /// <summary>
+    /// Runs a job that will be executed after the given <paramref name="delay"/>. The job will not be queued into the JobQueue, but executed directly.
+    /// The concurrency settings will be ignored.
+    /// </summary>
+    /// <param name="delay">The delay until the job will be executed.</param>
+    /// <param name="parameter">An optional parameter that is passed down as the <see cref="JobExecutionContext"/> to the job.</param>
+    /// <param name="token">An optional token to cancel the job.</param>
+    void ForceRunInstantJob<TJob>(TimeSpan delay, object? parameter = null, CancellationToken token = default)
```

Comment:
To stick to the naming - that should be `ForceRunScheduledJob`.
Also, here, the Minimal API would be nice (it can be followed up as well).

Also, we would need `ForceRunScheduledJob(DateTimeOffset absolute, ....)` just to have API symmetry.

Reply:
Done